README.md

```diff
@@ -2,6 +2,9 @@
 
 Smart ASA reference implementation combines the simplicity and security of the Algorand Standard Assets (ASA)s with the composability and programmability of Algorand Smart Contracts. The Smart ASA offers a new, powerful, L1 feature that extends regular ASAs up to the limits of your imagination!
 
+- [Smart ASA ABI JSON](https://github.com/algorandlabs/smart-asa/blob/main/smart_asa_abi.json)
+- [Smart ASA App TEAL](https://testnet.algoexplorer.io/application/107042851)
```

Comment:
So, we now have the SmartASA App deployed on testnes?

Reply:
Yes